clients/privacy-center/common/property-id.ts

```diff
@@ -0,0 +1,43 @@
+import { UserGeolocation } from "fides-js";
+import { NextApiRequest } from "next";
+import { PrivacyCenterEnvironment } from "~/app/server-environment";
+
+/**
+ * Verifies that certain conditions are met to be able to fetch experiences by property ID
+ */
+export const verifyConditionsForPropertyId = (
+  req: NextApiRequest,
+  geolocation: UserGeolocation | null,
+  environment: PrivacyCenterEnvironment,
+  fidesString: string | null
+) => {
+  const propertyId = req.query.property_id;
```

Comment:
Ooh, I thought of another nit! 😜

The fides-js API route has a detailed jsdoc comment that is used to general OpenAPI docs that document the supported headers/params/etc. Can you update that as well? Thanks 😊

Reply:
```
 *       - in: query
 *         name: property_id
 *         required: false
 *         description: Optional identifier used to filter for experiences associated with the given property. If omitted, returns all experiences not associated with any properties.
 *         schema:
 *           type: string
 ```